cli/actions/run.go

```diff
@@ -0,0 +1,22 @@
+package actions
+
+import (
+	"path/filepath"
+
+	"github.com/NethermindEth/sedge/configs"
+	"github.com/NethermindEth/sedge/internal/pkg/commands"
+)
+
+type RunContainersOptions struct {
+	GenerationPath string
+	Services       []string
+}
+
+func (s *sedgeActions) RunContainers(options RunContainersOptions) error {
+	upCmd := s.commandRunner.BuildDockerComposeUpCMD(commands.DockerComposeUpOptions{
+		Path:     filepath.Join(options.GenerationPath, configs.DefaultDockerComposeScriptName),
+		Services: options.Services,
+	})
+	_, err := s.commandRunner.RunCMD(upCmd)
```

Comment:
Sedge displays every command it runs to the user as a log info. At it very least it displays them as debug, but that's only for not very meaningful commands. The structure will look at this:
```suggestion
	upCmd := s.commandRunner.BuildDockerComposeUpCMD(commands.DockerComposeUpOptions{
		Path:     filepath.Join(options.GenerationPath, configs.DefaultDockerComposeScriptName),
		Services: options.Services,
	})
	log.Infof(configs.RunningCommand, upCmd.Cmd)
	_, err := s.commandRunner.RunCMD(upCmd)
```
DON'T COMMIT THIS SUGGESTION. This is only for illustration purposes, see that the log package is not being imported. I will refer at this change as "Log info the command" or "Log debug the command".

Reply:
Done https://github.com/NethermindEth/sedge/pull/203/commits/1f1e8620e142d2940d392b8674181bbab4066163

Reply:
Sorry if wasn't clear enough, in this case it is a log.info, as in the suggestion. The reason for that is that this is an important command and the users should be able to tell how to run the compose script in the same way Sedge did.

Reply:
Done https://github.com/NethermindEth/sedge/pull/203/commits/bfd0a19c7ac6b66afc1bf73ce4e43711bdc0d821